Tella/Data/Crypto/Cryptor.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  Cryptor.swift
+//  Tella
+//
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import CommonCrypto
+
+// Enum to handle encryption errors
+enum CryptoError: Error {
+    case cryptorCreationFailed
+    case encryptionFailed
+    case finalizationFailed
+}
+
+class Cryptor {
+    
+    static let bufferSize = 1024 * 1024 * 8  // 8 MB buffer size
```

Comment:
I have a question regrading the buffer size do we know why we are using this exact size ? because it depends on the nature of the Data -> If our data is composed of small chunks, using a smaller buffer size might be more efficient. On the other hand, if our data is typically larger, a larger buffer size could be beneficial.